litmus-portal/frontend/src/views/ChaosWorkflows/BrowseCluster/HeaderSection.tsx

```diff
@@ -0,0 +1,180 @@
+  const [state, setState] = useState([
+    {
+      startDate: new Date(),
+      endDate: new Date(),
+      key: 'selection',
+    },
+  ]);
+  return (
+    <div>
+      <div className={classes.headerSection}>
+        {/* Search Field */}
+        <InputBase
+          id="input-with-icon-adornment"
+          placeholder="Search"
+          className={classes.search}
+          value={searchValue}
+          onChange={changeSearch}
+          startAdornment={
+            <InputAdornment position="start">
+              <SearchIcon />
+            </InputAdornment>
+          }
+        />
+
+        {/* Select Workflow */}
+        <FormControl
+          variant="outlined"
+          className={classes.formControl}
+          color="secondary"
+          focused
+        >
+          <InputLabel className={classes.selectText}>Current Status</InputLabel>
+          <Select
+            value={statusValue}
+            onChange={changeStatus}
+            label="Workflow Status"
+            className={classes.selectText}
+          >
+            <MenuItem value="All">All</MenuItem>
+            <MenuItem value="Failed">Failed</MenuItem>
+            <MenuItem value="Running">Running</MenuItem>
+            <MenuItem value="Succeeded">Succeeded</MenuItem>
+          </Select>
+        </FormControl>
+
+        {/* Select Cluster */}
+        <FormControl
+          variant="outlined"
+          className={classes.formControl}
+          color="secondary"
+          focused
+        >
+          <InputLabel className={classes.selectText}>Target Cluster</InputLabel>
+          <Select
+            value={clusterValue}
+            onChange={changeCluster}
+            label="Target Cluster"
+            className={classes.selectText}
+          >
+            <MenuItem value="All">All</MenuItem>
+            <MenuItem value="Internal">Internal</MenuItem>
+            <MenuItem value="External">External</MenuItem>
+          </Select>
+        </FormControl>
+
+        <ButtonOutline isDisabled={false} handleClick={popOverClick}>
+          <Typography className={classes.displayDate}>
+            {displayDate}
+            <IconButton className={classes.iconButton}>
+              {isOpen ? <KeyboardArrowDownIcon /> : <ChevronRightIcon />}
+            </IconButton>
+          </Typography>
+        </ButtonOutline>
+        <Popover
+          open={isOpen}
+          anchorEl={popAnchorEl}
+          onClose={popOverClose}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'center',
+          }}
+          transformOrigin={{
+            vertical: 'top',
+            horizontal: 'center',
+          }}
+          className={classes.popOver}
+        >
+          <DateRangePicker
+            onChange={(item) => {
+              setState([(item as any).selection]);
+              selectDate(
+                `${(item as any).selection.startDate}`,
+                `${(item as any).selection.endDate}`
```

Comment:
Can we have a proper type for these instead of `any`

Reply:
I think it's fine for now @S-ayanide , We tried to add the types earlier but we got some errors.